tests/fixtures/foos-bar-baz-remove-route.js

```diff
@@ -1,5 +1,5 @@
 Router.map(function() {
-  this.resource('foos', function() {
+  this.route('foos', function() {
```

Comment:
I think we can remove all files related with resources and have the proper tests for routes